File: SimulasiAPBN.Web/React/src/core/types/ClientSimulation.ts
/**
 * Simulasi APBN
 *
 * Program ditulis oleh Danang Galuh Tegar Prasetyo (https://danang.id/)
 * untuk Kementerian Keuangan Republik Indonesia.
 */
import Allocation from "@core/models/Allocation"
import SimulationSession from "@core/models/SimulationSession"
import EconomicMacro from "@core/models/EconomicMacro"

type ClientSimulation = {
	allocations: Allocation[]
	economics: EconomicMacro[]
	session: SimulationSession
}

export default ClientSimulation

File: SimulasiAPBN.Web/Models/ClientSimulation.cs
using SimulasiAPBN.Core.Models;
using System.Collections.Generic;

namespace SimulasiAPBN.Web.Models
{
    public class ClientSimulation
    {

        public IEnumerable<Allocation> Allocations { get; set; }
        public IEnumerable<EconomicMacro> Economics { get; set; }

        public SimulationSession Session { get; set; }

    }
}
